lib/api.js

```diff
@@ -14,9 +14,8 @@ import _ from 'lodash';
 var api = new base.API({
   title:        "TaskCluster Secrets API Documentation",
   description: [
-    "The secrets service, typically available at",
-    "`tools.taskcluster.net`, is responsible for managing",
-    "secure data in TaskCluster."
```

Comment:
Note, the _typically available at_ line appears in most of our services - we should probably standardise whether we keep it or discard it.

e.g.

* [queue](https://github.com/taskcluster/taskcluster-queue/blob/e03410b5f278f16d9d0080a58cb5bc05fdab7147/routes/v1.js#L79)
* [scheduler](https://github.com/taskcluster/task-graph-scheduler/blob/1066d15d0aee6870b974f9b122ca9dbec31a1ad7/routes/api/v1.js#L30)
* [index](https://github.com/taskcluster/taskcluster-index/blob/95943c951566819128f91811e3c91b66457c5dfd/routes/api/v1.js#L23)

Admittedly, this should probably say `secrets.taskcluster.net` rather than `tools.taskcluster.net`.

If we do keep it in, we should probably also get the API docs to be shown if you visit the domain root (e.g. secrets.taskcluster.net) rather than returning `Cannot GET /` as is currently the case.

@djmitche what do you think about this idea?